copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,857 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    valid_pipe.metadata["metadata"]["filters.stats"]["statistic"]
</pre><p>Should "statistic" be moved down to "Accessing Data" and the suggestion from the above comment be swapped in here? or is this saying that we can use the statistics to check validation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>The "statistic" is shown to show the stat from a copc file and is not related to validation. </p>